### PR TITLE
Add make my book button functionality

### PR DESCRIPTION
**Pull request by stephanie-roe**
# Pull Request Template 

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

-Added code to allow the "make my cover" button to work properly , as well as display the user-created cover on the home page, in accordance with all bulleted steps of Iteration 3 on the Romcom Rubric.

Fixes # (issue)

## Type of Change:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Refactor (code is working and without bugs, formatting has been streamlined)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

- [x] Have you used Dev Tools/ ran code in the console?
- [x] Have you verified changes using `open index.html`?

## Checklist:

- [x] My code follows the style guidelines of this project.
- [x] My commit messages are consistent, descriptive, concise and begin with a verb and capital letter.
- [x] I have performed a self-review of my own code.
- [x] I have added @jlavarine and @stephanie-roe as reviewers.

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -65,4 +71,21 @@ function clickHomeButton() {
   document.querySelector('.random-cover-button').classList.remove('hidden');
   document.querySelector('.home-button').classList.add('hidden');
   document.querySelector('.form-view').classList.add('hidden');
+  displayCustomCover(currentCover);
+};
+
+function clickMakeMyBookButton() {
```
Comment (rachael-t) [inline src/main.js]: What could be a more descriptive name for the action that this function is doing? The name of the function makes me think that what the function is going to do is the actual clicking of the button.

File: src/main.js [hunk 2]
```diff
@@ -65,4 +71,21 @@ function clickHomeButton() {
...
+  covers.push(coverInput.value);
+  titles.push(titleInput.value);
+  descriptors.push(descriptor1Input.value);
+  descriptors.push(descriptor2Input.value);
+  currentCover = new Cover(coverInput.value, titleInput.value, descriptor1Input.value, descriptor2Input.value);
+  event.preventDefault();
```
Comment (rachael-t) [inline src/main.js]: Does this make sense having it be the last thing done in this function?

File: src/main.js [hunk 3]
```diff
@@ -65,4 +71,21 @@ function clickHomeButton() {
...
   document.querySelector('.form-view').classList.add('hidden');
+  displayCustomCover(currentCover);
+};
+
+function clickMakeMyBookButton() {
+  covers.push(coverInput.value);
+  titles.push(titleInput.value);
+  descriptors.push(descriptor1Input.value);
+  descriptors.push(descriptor2Input.value);
+  currentCover = new Cover(coverInput.value, titleInput.value, descriptor1Input.value, descriptor2Input.value);
```
Comment (rachael-t) [inline src/main.js]: Lines 78 - 81 are doing one thing, and line 82 is doing another. Could you possibly separate some of this out into another function?

**Comment by rachael-t** [comment]
Great job with this so far! 🎉

Looking over the iteration 3 bullet points and testing the site there are a few things to consider. When I click the "Make My Own Book" button, I do not receive any indication as to whether or not it was successful. Do you want to display a message (i.e. "Success!")? Redirect the user to see the book cover that was created? Also, something to think about is after a user adds information to the input fields and clicks the button, do you want to clear those fields out so that they can create another?

Let me know if you have any questions!

**Review by Jlavarine — APPROVED** [review]
Looks great! I like the idea of renaming the "clickMakeMyBookButton" to  "addUserValue".

**Pull request merged**